Shared/dugga.js

```diff
@@ -1017,13 +1017,27 @@ function AJAXService(opt,apara,kind)
 	}
 
 	if(kind=="COURSE"){
+		// Call microservice to create new course 
+		// TODO: This is probably not a very good solution, this is done with the opt comparison
+		// For now this is usable, but should be remade when all microservices related 
+		// to courseedservice.php are implemented
+		if(opt === "NEW"){
```

Comment:
The problem is that our microservice should only create a new course, not get. When you start courseed.php coursedservice.php opt=GET. This will fetch all courses. Our service as said should not do that, only opt=NEW. Thats why in dugga.js we have an if statement so that the service knows when to use our microservice.